### DISCUSSION TITLE
Show and hidden Text

### DISCUSSION THREAD

**Discussion by deki84** [neuefische/questions#164] [Web] [answered]
Hello everyone. I have a problem. I've been searching and googling for a while, but unfortunately I can't find the answer. I would like my text to be visible only when I press the Click me button. 
<img width="415" alt="Bildschirmfoto 2022-09-23 um 13 19 31" src="https://user-images.githubusercontent.com/35610295/191949604-034ee16a-a117-4517-8f2e-341e60dc3814.png">
<img width="635" alt="Bildschirmfoto 2022-09-23 um 13 20 35" src="https://user-images.githubusercontent.com/35610295/191949779-a098808b-b697-4425-943c-30dfe6012d34.png">
<img width="610" alt="Bildschirmfoto 2022-09-23 um 13 20 59" src="https://user-images.githubusercontent.com/35610295/191949847-ee8fa2f1-bd93-4cdf-bd45-75df620670e9.png">

**Comment by K-Juno** [comment, marked as answer]
Hi Deki! I am working on a similar case, have you tried conditional rendering?
For instance:
{value && <Text>Your current value is {value}</Text>}
Not sure, whether that would work for you, but you might check out:
https://reactjs.org/docs/conditional-rendering.html
Hope this helps!!

**Reply by deki84** [reply]
Thanks Juni, I have found the code after all. Here you can see how it is done.
https://codesandbox.io/s/dl55j?file=/src/App.js:41-284
<img width="606" alt="Bildschirmfoto 2022-09-23 um 15 43 00" src="https://user-images.githubusercontent.com/35610295/191973880-858f5de4-6ca7-4e9f-83bd-9ffd1191daa1.png">
